### ISSUE TITLE
Main scss file not compiling when saving a partial file 

### ISSUE THREAD

**Issue by blazicke** [Wscats/compile-hero#69]
Hi, I have a main.scss file which imports (@use) a whole bunch of partials. Is there a way to compile it each time I update and save one of this partials?

Thanks!

**Comment by Wscats** [comment]
> Hi, I have a main.scss file which imports (@use) a whole bunch of partials. Is there a way to compile it each time I update and save one of this partials?
> 
> Thanks!

You can put some files in the folder, and then use `Compile File(s)` menu item against the folder.

![image](https://user-images.githubusercontent.com/17243165/95283118-9188ba80-088d-11eb-9ff8-13dff25fdac6.png)

**Comment by blazicke** [comment]
Yep, but I have to do that every time I update one of the files (quite annoying). 
Is there a way to compile a specific file when I edit one of its partials?

Thaks!

**Comment by revenz** [comment]
+1 to this.   its an annoying thing to have to do.
